### DISCUSSION TITLE
How to solve an LMI

### DISCUSSION THREAD

**Discussion by ayyobasa** [yalmip/YALMIP#981] [General discussion] [answered]
Dear Johan,

Hello 

Here I have an LMI which I do not know how to solve it 

Is there any sample code that I can study and learn how to deal with it?

Here both P>=0 and lambda are unknown and Pi >=0 matrices are tunable.

I would be so thankful if you guide me, please.

Thanks in advance.

![2222222222222211111111111eeeeeeeeeeeeeeeeeeeeeeeeeee](https://user-images.githubusercontent.com/87689237/132009729-62ac6889-99c5-4196-bf89-614595f54559.PNG)

**Comment by johanlofberg** [comment, marked as answer]
It's not an LMI as you have unsymmetric terms. Your derivation of that condition is severely wrong.  I think we have discussed that already elsewhere.

**Reply by ayyobasa** [reply]
Dear Johan

yes sorry, it seems I made a typing latex error.

The right format is as follow

**Reply by johanlofberg** [reply]
It's still massively unsymmetric

**Reply by ayyobasa** [reply]
Thank u so much here P is symmetric. Is there a text or reference that can help me?

**Comment by johanlofberg** [comment]
PB-Pi_4, Psi_1, Psi_2  etc are not symmetric

I can only recommend you to read any introductory text on LMIs and semidefinite programming, in particular its use in control as you've completely messed up your stability proof https://web.stanford.edu/~boyd/lmibook/lmibook.pdf
